products/google-cloud-sql-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+# Name of the product (mandatory).
+title: Cloud SQL Auth Proxy
+category: service
+tags: database google server-app service
+iconSlug: googlecloud
+permalink: /cloud-sql-auth-proxy
```

Comment:
The markdown file must be renamed after the permalink : `cloud-sql-auth-proxy.md`.